Test/Pester5.Tests.ps1

```diff
@@ -320,102 +219,26 @@ Describe 'Passing' {
         $true | Should -BeTrue
     }
 }
-'@
-        WhenPesterTaskIsInvoked -AsJob -ErrorAction SilentlyContinue
-        ThenPesterShouldHaveRun -FailureCount 1 -PassingCount 1
-    }
-}
-
-Describe 'Pester5.when run multiple times in the same build' {
-    AfterEach { Reset }
-    It 'should run multiple times' {
-        Init
-        GivenTestFile 'PassingTests.ps1' @'
-Describe 'PassingTests' {
-    It 'should pass' {
-        $true | Should -BeTrue
-    }
-}
-'@
-        WhenPesterTaskIsInvoked -AsJob
-        WhenPesterTaskIsInvoked -AsJob
-        ThenPesterShouldHaveRun -PassingCount 2 -FailureCount 0
-        ThenTestShouldCreateMultipleReportFiles
-    }
-}
-
-Describe 'Pester5.when missing path' {
-    AfterEach { Reset }
-    It 'should fail' {
-        Init
-        WhenPesterTaskIsInvoked -AsJob -ErrorAction SilentlyContinue
-        ThenPesterShouldHaveRun -PassingCount 0 -FailureCount 0
-        # ThenTestShouldFail -failureMessage 'Property "Script": Script is mandatory.'
-    }
-}
-
-Describe 'Pester5.when a task path is absolute' {
-    AfterEach { Reset }
-    It 'should fail' {
-        Init
-        $pesterPath = Join-Path -Path $testRoot -ChildPath '..\SomeFile'
-        New-Item -Path $pesterPath
-        GivenTestFile $pesterPath
-        WhenPesterTaskIsInvoked -AsJob -ErrorAction SilentlyContinue
-        ThenPesterShouldHaveRun -PassingCount 0 -FailureCount 0
-        # ThenTestShouldFail -failureMessage 'outside\ the\ build\ root'
-    }
-}
-
-Describe 'Pester5.when excluding tests and an exclusion filter doesn''t match' {
-    AfterEach { Reset }
-    It 'should still run' {
-        Init
-        GivenTestFile 'PassingTests.ps1' @'
-Describe 'PassingTests' {
-    It 'should pass' { 
-        $true | Should -BeTrue
-    }
-}
-'@
-        GivenTestFile 'FailingTests.ps1' @'
-Describe 'FailingTests' {
-    It 'should fail' {
-        $true | Should -BeFalse
-    }
-}
-'@
-        WhenPesterTaskIsInvoked -AsJob -WithArgument @{
-            'Exclude' = '*fail*','Passing*'
-        }
-        ThenPesterShouldHaveRun -FailureCount 0 -PassingCount 1
-    }
-}
-
-Describe 'Pester5.when excluding tests and exclusion filters match all paths' {
-    AfterEach { Reset }
-    It 'should fail' {
-        Init
-        GivenTestFile 'PassingTests.ps1' @'
-Describe 'PassingTests' {
-    It 'should pass' {
-        $true | Should -BeTrue
-    }
-}
-'@
-        GivenTestFile 'FailingTests.ps1' @'
-Describe 'FailingTests' {
-    It 'should fail' {
-        $true | Should -BeFalse
-    }
-}
 '@
         WhenPesterTaskIsInvoked -AsJob -WithArgument @{
-            'Exclude' = (Join-Path -Path '*' -ChildPath 'Fail*'),(Join-Path -Path '*' -ChildPath 'Passing*')
+            Debug = @{

```

Comment:
I think you just need `Run.Path` since you're only testing that the task passes.